feat(curriculum): add cipher project to Python curriculum (#52373)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/20-upcoming-python/learn-string-manipulation-by-building-a-cipher/6554930320d70414e7b6acc6.md

---
id: 6554930320d70414e7b6acc6
title: Step 54
challengeType: 20
dashedName: step-54
---

# --description--

> Delete your `shift` variable. Then create another variable called `custom_key` and give it the value of the string `python`.

Reviewer: ```suggestion
Delete your `shift` variable. Then, declare a new variable called `custom_key` and assign the value of the string `python` to this variable.
```

Reviewer: I would suggest updating "create" to "declare" for consistency with the hint.
